.buildkite/scripts/steps/renovate.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo '--- Renovate: validation'
+.buildkite/scripts/steps/checks/renovate.sh
```

Comment:
I think this script didn't get pushed

Reply:
@Ikuni17 this isn't a new file, but rather an existing script.